recipes/tmap/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+cd tmap/
+$PYTHON setup.py install
```

Comment:
I didn't check the `setup.py`, but if it doesn't do any custom things, you could use
```sh
"${PYTHON}" -m pip install . --no-deps -vv
```
here.
(Or `script: {{ PYTHON }} -m pip install . --no-deps -vv` in `meta.yaml`.)